pkg/volume/csi/csi_plugin.go

```diff
@@ -907,3 +1014,15 @@ func isV1Version(version string) bool {
 
 	return parsedVersion.Major() == 1
 }
+
+// ToPluginHandler converts the generic volume plugin to a specific csiPlugin and
+// returns that as a PluginHandler. In case the generic plugin is not a CSI
+// plugin and thus not a PluginHandler, an error is returned.
```

Comment:
Please indicate in comment that this is used by pluginwatcher as a way to get access to the csiPlugin.